packages/php/email-editor/src/Integrations/Core/theme.json

```diff
@@ -38,6 +38,24 @@
 				},
 				"css": "&.has-text-align-right { border-width: 0 1px 0 0; } &.has-text-align-center { border-width: 0; border-inline: 0; padding-inline: 0; }",
 				"variations": {}
+			},
+			"core/table": {
+				"spacing": {
+					"margin": {
+						"top": "0",
+						"bottom": "0"
+					},
+					"padding": {
+						"top": "10px",
```

Comment:
I noticed that this padding is only used in the editor, resulting in styles mismatch. Is it really needed?